secscan/Jenkinsfile

```diff
@@ -57,8 +57,8 @@ def notifyBuild(String buildStatus = 'STARTED') {
   } else {
     color = 'RED'
     colorCode = '#FF0000'
+    // Send notifications
+    slackSend (color: colorCode, message: summary, channel: 'C01G91NUHFY',  teamDomain: 'rancher',  tokenCredentialId: 'longhorn-tests-slack-token')
   }
 
-  // Send notifications
```

Comment:
Still keep this, otherwise, it will look weird since a test has started but you won't see it finish.

Reply:
fixed